fxa/oauth.py

```diff
@@ -239,11 +240,21 @@ def verify_token(self, token, scope=None):
             resp = None
 
         if resp is None:
-            url = '/verify'
-            body = {
-                'token': token
+            ## TODO we want to fetch https://oauth.accounts.firefox.com/.well-known/openid-configuration and then get the jwks_uri key to get the following url, but we'll just hardcode it like this for now until we are closer to production. We need to do this because oauth.accounts.firefox.com does not yet respond to .well-known, but accounts.firefox.com does; for consistency, oauth.accounts.firefox.com should respond to .well-known. 
+            
+            secret = jwt.algorithms.RSAAlgorithm.from_jwk(
+                self.apiclient.get('/v1/jwks')
+            )
+            decoded = jwt.decode(
+                token, secret, algorithms=['RS256'], options={'verify_aud': False}
```

Comment:
nit: we probably need a comment here to justify the use of `verify_aud: False`. Something like "The FxA OAuth ecosystem currently doesn't make good use of `aud`, and instead relies on `scope` for restricting which services can accept which tokens. So there's no value in checking it here."

Reply:
I'm just stealing your comment verbatim :)